src/Case-ABL.f90

```diff
@@ -120,20 +120,20 @@ subroutine init_abl(ux1,uy1,uz1,ep1,phi1)
         endif
       enddo
 
-      ! Add random noise
-      do j=1,xsize(2)
-        if (istret==0) y=real(j + xstart(2)-1-1,mytype)*dy
-        if (istret/=0) y=yp(j+xstart(2)-1)
-        !if (y.lt.50) then 
-        !  do k=1,xsize(3)
-        !  do i=1,xsize(1)
-        !    call random_number(phinoise)
-        !    phinoise=0.1*(phinoise*2.-1.)
-        !    phi1(i,j,k,1)=phi1(i,j,k,1)+phinoise
-        !  enddo
-        !  enddo
-        !endif
-      enddo
+      !! Add random noise
```

Comment:
This could be removed ?

Reply:
No idea if useful or not. We should ask to someone that uses the ABL case to see if the noise for Phi is necessary or not

Reply:
Why this part of the code has been changed? This is not related to MHD

Reply:
I have removed the random noise since it was not used. However, just to be clear this was already here before and it has not been added with the MHD module